iamaccessgroupsv2/iam_access_groups_v2_examples_test.go

```diff
@@ -614,5 +620,525 @@ var _ = Describe(`IamAccessGroupsV2 Examples Tests`, func() {
+				Users: []string{"IBMid-50PJGPKYJJ", "IBMid-665000T8WY"},
+				ActionControls: membersActionControlsModel,
+			}
+
+			conditionInputModel := &iamaccessgroupsv2.ConditionInput{
+				Claim: core.StringPtr("blueGroup"),
+				Operator: core.StringPtr("CONTAINS"),
+				Value: core.StringPtr(`"test-bluegroup-saml"`),
+			}
+
+			rulesActionControlsModel := &iamaccessgroupsv2.RulesActionControls{
+				Remove: core.BoolPtr(false),
+				Update: core.BoolPtr(false),
+			}
+
+			ruleInputModel := &iamaccessgroupsv2.RuleInput{
+				Name: core.StringPtr("Manager group rule"),
+				Expiration: core.Int64Ptr(int64(12)),
+				RealmName: core.StringPtr("https://idp.example.org/SAML2"),
+				Conditions: []iamaccessgroupsv2.ConditionInput{*conditionInputModel},
+				ActionControls: rulesActionControlsModel,
+			}
+
+			assertionsActionControlsModel := &iamaccessgroupsv2.AssertionsActionControls{
+				Add: core.BoolPtr(false),
+				Remove: core.BoolPtr(true),
+				Update: core.BoolPtr(true),
+			}
+
+			assertionsInputModel := &iamaccessgroupsv2.AssertionsInput{
+				Rules: []iamaccessgroupsv2.RuleInput{*ruleInputModel},
+				ActionControls: assertionsActionControlsModel,
+			}
+
+			accessActionControlsModel := &iamaccessgroupsv2.AccessActionControls{
+				Add: core.BoolPtr(false),
+			}
+
+			groupActionControlsModel := &iamaccessgroupsv2.GroupActionControls{
+				Access: accessActionControlsModel,
+			}
+
+			accessGroupInputModel := &iamaccessgroupsv2.AccessGroupInput{
+				Name: core.StringPtr("IAM Admin Group"),
+				Description: core.StringPtr("This access group template allows admin access to all IAM platform services in the account."),
+				Members: membersInputModel,
+				Assertions: assertionsInputModel,
+				ActionControls: groupActionControlsModel,
+			}
+
+			policyTemplatesInputModel := &iamaccessgroupsv2.PolicyTemplatesInput{
+				ID: &testPolicyTemplateID,
+				Version: core.StringPtr("1"),
+			}
+
+			createTemplateOptions := iamAccessGroupsService.NewCreateTemplateOptions(
+				"IAM Admin Group template",
+				testAccountID,
+			)
+			createTemplateOptions.SetDescription("This access group template allows admin access to all IAM platform services in the account.")
+			createTemplateOptions.SetGroup(accessGroupInputModel)
+			createTemplateOptions.SetPolicyTemplateReferences([]iamaccessgroupsv2.PolicyTemplatesInput{*policyTemplatesInputModel})
+
+			createTemplateResponse, response, err := iamAccessGroupsService.CreateTemplate(createTemplateOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(createTemplateResponse, "", "  ")
+			fmt.Println(string(b))
+			// end-create_template
+			testTemplateId = *createTemplateResponse.ID
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(createTemplateResponse).ToNot(BeNil())
+		})
+		It(`ListTemplates request example`, func() {
+			fmt.Println("\nListTemplates() result:")
+			// begin-list_templates
```

Comment:
Please add a new line after the `// begin-...` tag.
```suggestion
			// begin-list_templates

```